examples/simple-scenario/.gitignore

```diff
@@ -0,0 +1,84 @@
+# Created by .ignore support plugin (hsz.mobi)
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion
+
+../../.idea/modules/akkadebugging-core-build.iml
```

Comment:
But I think we can leave it for now - it works. We can tidy up .gitignore later. 
(Very big part of this file is automatically generated - so maybe it's connected with that.)